libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -19,13 +19,108 @@
+#endif  // defined(MAV_SYSID_DEFAULT)
+
+const AP_Param::GroupInfo GCS::var_info[] {
+    // @Param: _SYSID
+    // @DisplayName: MAVLink system ID of this vehicle
+    // @Description: Allows setting an individual MAVLink system id for this vehicle to distinguish it from others on the same network
+    // @Range: 1 255
+    // @User: Advanced
+    AP_GROUPINFO("_SYSID",    1,     GCS,  sysid,  MAV_SYSID_DEFAULT),
+
+    // @Param: _MYGCS_SYSID
+    // @DisplayName: My ground station number
+    // @Description: This controls whether packets from other than the expected GCS system ID will be accepted
+    // @Range: 1 255
+    // @Increment: 1
+    // @User: Advanced
+    AP_GROUPINFO("_MYGCS_SYSID",    2,      GCS, mav_mygcs_sysid, 255),
+
+    // @Param: _MYGCS_ENFRCE
+    // @DisplayName: GCS sysid enforcement
+    // @Description: Allows restricting radio overrides to only come from my ground station
+    // @Values: 0:NotEnforced,1:Enforced
+    // @User: Advanced
+    AP_GROUPINFO("_MYGCS_ENFRCE",    3,      GCS, mav_mygcs_enforce, 0),
+
+    // @Param: _TELEM_DELAY
+    // @DisplayName: Telemetry startup delay
+    // @Description: The amount of time (in seconds) to delay radio telemetry to prevent an Xbee bricking on power up
+    // @User: Advanced
+    // @Units: s
+    // @Range: 0 30
+    // @Increment: 1
+    AP_GROUPINFO("_TELEM_DELAY",    4,      GCS, mav_telem_delay, 0),
+
+#if MAVLINK_COMM_NUM_BUFFERS > 0
+    // @Group: 1
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[0], "1", 11, GCS, _chan_var_info[0]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 1
+    // @Group: 2
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[1], "2", 12, GCS, _chan_var_info[1]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 2
+    // @Group: 3
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[2], "3", 13, GCS, _chan_var_info[2]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 3
+    // @Group: 4
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[3], "4", 14, GCS, _chan_var_info[3]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 4
+    // @Group: 5
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[4], "5", 15, GCS, _chan_var_info[4]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 5
+    // @Group: 6
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[5], "6", 16, GCS, _chan_var_info[5]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 6
+    // @Group: 7
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[6], "7", 17, GCS, _chan_var_info[6]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 7
+    // @Group: 8
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[7], "8", 18, GCS, _chan_var_info[7]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 8
+    // @Group: 9
+    // @Path: GCS_MAVLink_Parameters.cpp
+    AP_SUBGROUPVARPTR(_chan[8], "9", 19, GCS, _chan_var_info[8]),
+#endif
+#if MAVLINK_COMM_NUM_BUFFERS > 9
```

Comment:
let's increase MAVLINK_COMM_NUM_BUFFERS to 10 for all targets I think, it costs almost nothing

Reply:
Discussed on DevCall.  Needs testing, so in a future PR.